doc/nrf/samples.rst

```diff
@@ -19,6 +19,15 @@ General information about samples in the |NCS|
      You can change the default behavior by updating the configuration option :kconfig:option:`CONFIG_RESET_ON_FATAL_ERROR`.
    * All samples in the |NCS| are tested and verified in accordance with their :ref:`maturity level <software_maturity>`.
 
+.. samples_sample_helper_start
+
+.. note::
+   Samples found in the :file:`samples/common` are not intended to be used independently, but in addition to other compatible samples for development purposes.
+   For example, the file path :file:`samples/common/mcumgr_bt_ota_dfu` contains the source for the :kconfig:option:`CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU` Kconfig option.
+   This Kconfig option can be used with specific Nordic Semiconductor devices to enable MCUmgr to perform firmware over-the-air (FOTA) updates using Bluetooth® Low Energy.
+
+.. samples_sample_helper_end
```

Comment:
For this part: 

1. I would remove the include tags:
```suggestion
.. note::
   Samples found in the :file:`samples/common` are not intended to be used independently, but in addition to other compatible samples for development purposes.
   For example, the file path :file:`samples/common/mcumgr_bt_ota_dfu` contains the source for the :kconfig:option:`CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU` Kconfig option.
   This Kconfig option can be used with specific Nordic Semiconductor devices to enable MCUmgr to perform firmware over-the-air (FOTA) updates using Bluetooth® Low Energy.
```

2. And then move the text to shortcuts.txt and include it in the note as a shortcut. This way, you can still use it in advanced_building.rst.
```suggestion
.. note::
   |common_sample_components_desc|
```

Reply:
Good tip - will implement!